webpack.config.ts

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  entry: './src/cli.ts',
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: 'ts-loader',
+        exclude: /node_modules/,
+      },
+      {
+        test: /\.node$/,
```

Comment:
what is it used for? just curious

Reply:
we have a .node compiled package which needs to be loaded

Reply:
yes, but what package it is? fsevents?

Reply:
yep

Reply:
can it be null-loaded too?